cli/endpoints/online/triton/single-model/single-triton-lite.ipynb

```diff
@@ -0,0 +1,291 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: azure-storage-blob in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (12.8.1)\n",
+      "Requirement already satisfied: azure-core<2.0.0,>=1.10.0 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from azure-storage-blob) (1.13.0)\n",
+      "Requirement already satisfied: cryptography>=2.1.4 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from azure-storage-blob) (3.4.7)\n",
+      "Requirement already satisfied: msrest>=0.6.18 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from azure-storage-blob) (0.6.21)\n",
+      "Requirement already satisfied: requests>=2.18.4 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from azure-core<2.0.0,>=1.10.0->azure-storage-blob) (2.25.1)\n",
+      "Requirement already satisfied: six>=1.11.0 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from azure-core<2.0.0,>=1.10.0->azure-storage-blob) (1.15.0)\n",
+      "Requirement already satisfied: cffi>=1.12 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from cryptography>=2.1.4->azure-storage-blob) (1.14.5)\n",
+      "Requirement already satisfied: pycparser in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from cffi>=1.12->cryptography>=2.1.4->azure-storage-blob) (2.20)\n",
+      "Requirement already satisfied: isodate>=0.6.0 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from msrest>=0.6.18->azure-storage-blob) (0.6.0)\n",
+      "Requirement already satisfied: certifi>=2017.4.17 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from msrest>=0.6.18->azure-storage-blob) (2020.12.5)\n",
+      "Requirement already satisfied: requests-oauthlib>=0.5.0 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from msrest>=0.6.18->azure-storage-blob) (1.3.0)\n",
+      "Requirement already satisfied: idna<3,>=2.5 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from requests>=2.18.4->azure-core<2.0.0,>=1.10.0->azure-storage-blob) (2.10)\n",
+      "Requirement already satisfied: chardet<5,>=3.0.2 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from requests>=2.18.4->azure-core<2.0.0,>=1.10.0->azure-storage-blob) (4.0.0)\n",
+      "Requirement already satisfied: urllib3<1.27,>=1.21.1 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from requests>=2.18.4->azure-core<2.0.0,>=1.10.0->azure-storage-blob) (1.26.4)\n",
+      "Requirement already satisfied: oauthlib>=3.0.0 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from requests-oauthlib>=0.5.0->msrest>=0.6.18->azure-storage-blob) (3.1.0)\n"
```

Comment:
Include output only where it's helpful to the user.  Verbose output like this should be removed / cleaned up throughout the notebook.

Reply:
Done